src/defaultsettings.cpp

```diff
@@ -97,7 +97,17 @@ void set_default_settings()
 	// Client
 	settings->setDefault("address", "");
 	settings->setDefault("enable_sound", "true");
+#if defined(__linux__) && !defined (__ANDROID__)
```

Comment:
`defined(__unix__) && !defined(__APPLE__) && !defined (__ANDROID__)`
might be better because BSD also uses X11